wagtail/admin/api/actions/revert_to_page_revision.py

```diff
@@ -0,0 +1,44 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.api.v2.utils import BadRequestError
+from wagtail.core.actions.revert_to_page_revision import (
+    RevertToPageRevisionAction,
+    RevertToPageRevisionError,
+)
+
+from .base import APIAction
+
+
+class RevertToPageRevisionAPIActionSerializer(Serializer):
+    previous_revision_id = fields.IntegerField()
```

Comment:
Maybe name this to ``revision_id`` (I think the ``previous_`` bit is implied from the action)